client/components/auth/user.service.js

```diff
@@ -31,6 +31,13 @@ function UserResource($resource) {
         controller:'events'
       },
       isArray: true
+    },
+    getFullSections: {
+      method: 'GET',
+      params: {
+        controller:'instructor',
+        options: 'fullsections'
+      }
```

Comment:
these should be changed so accessing the index with GET parameters yields the sections if there's a `withSections` param